pkg/util/testingjobs/pod/wrappers.go

```diff
@@ -135,6 +137,15 @@ func (p *PodWrapper) KueueSchedulingGate() *PodWrapper {
 	return p
 }
 
+// TopologySchedulingGate adds kueue scheduling gate to the Pod
+func (p *PodWrapper) TopologySchedulingGate() *PodWrapper {
+	if p.Spec.SchedulingGates == nil {
```

Comment:
```suggestion
	if len(p.Spec.SchedulingGates) == 0 {
```

Reply:
Actually, the `if` is not needed at all. Dropped, and used the `Gate` function instead.